javascript/javascript3/week2/app.js

```diff
@@ -0,0 +1,69 @@
+let exchangeRates;
+
+const getExchangeData = async () => {
+  const res = await fetch("https://open.er-api.com/v6/latest/USD");
+  const data = await res.json();
+  exchangeRates = data.rates;
+  const currenciesList = Object.keys(exchangeRates);
```

Comment:
please read about hoisting concept in js. also see either we should define variable in start of function or it work anywhere.